src/styles/header.scss

```
@@ -0,0 +1,77 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05%);
+}
+
+.is-active {
+  position: relative;
+  padding: 23px 0;
+  font-size: 12px;
+  margin-right: 20px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #060b35;
+  cursor: pointer;
+  font-family: inherit;
+  transition: color 0.3s ;
+  font-weight: 500;
+
+  &::after {
+    display: block;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    content: "";
+    background-color: #00acdc;
+    height: 4px;
+    width: 100%;
+    border-radius: 8px;
+    margin-bottom: 0;
+  }
+}
+
+.nav {
+  &:hover,
+  .is-active {
+    color: #00acdc;
+  }
+
+  &:nth-child(8) {
+    margin-right: 0;
+  }
+}
+
+.nav-list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+.nav-item {
+  display: flex;
+  position: relative;
+}
```

Use contribution for all child selectors's classes.
```suggestion
.nav {
  &:hover,
  .is-active {
    color: #00acdc;
  }

  &:nth-child(8) {
    margin-right: 0;
  }
  &-list {
    display: flex;
    margin: 0;
    padding: 0;
    list-style: none;
    box-sizing: border-box;
  }

  &-item {
    display: flex;
    position: relative;
  }
}

```